vendor/gopkg.in/fsnotify.v1/.gitignore

```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This isn't technically our code since it's a vendored dependency -- are we allowed to add the license here?

Reply:
Nope. Good catch. We shouldn't be touching vendored stuff, other people's code has its own licensing documentation in LICENSE.

Moreover, touching vendored stuff runs the risk of creating a Derivative Work, which has its own legal and copyright implications and we may or may not have the rights to do. So we should avoid modifying files we don't own if at all possible.

Reply:
My mistake, I meant to exclude all vendored files from the gitignore commit, I must have missed one. Needless to say, Weasel doesn't mind if there's no license there. Removed in 2f75f3a3fa.